README.md

```diff
@@ -48,6 +48,8 @@ tools/debian_buildenv.sh setup` respectively), then run:
 There should now be a `mixxx` executable in the current directory that you can
 run. Alternatively, can generate a package using `cpack`.
 
+*Note: Using Powershell may be problematic. If you are having trouble building the .sln from a Windows environment (i.e. cmake FindChromaprint) try opening CMD and launching the build script `.\tools\windows_buildenv.bat` from the command line.*
```

Comment:
This is unneeded IMHO. We explicitly mentioned what command prompt should be used (i.e. the "x64 Native Tools Command Prompt for VS 2019", not Powershell) a few lines above.

I also don't understand what the `.sln`file  or the `FindChromaprint` module have to do with this?